server/lib/chat/chat-models.ts

```diff
@@ -116,6 +117,41 @@ export async function getUserChannelEntriesForUser(
   }
 }
 
+export async function countUserJoinedChannels(
+  userId: SbUserId,
+  withClient?: DbClient,
+): Promise<number> {
+  const { client, done } = await db(withClient)
+  try {
+    const result = await client.query<DbUserChannelEntry>(sql`
+      SELECT *
+      FROM channel_users
+      WHERE user_id = ${userId};
+    `)
+    return result.rowCount
```

Comment:
I've already recommended to delete all this code, but for future reference: uh what? Why are you using `rowCount` to perform a `COUNT(*)`? The database can potentially optimize the `COUNT`, especially since it operates only on indexes, but instead you wrote a query to return a bunch of data and then immediately throw it all away? Please don't ever do this